public/src/components/epicTests/epicTestVariantEditor.tsx

```diff
@@ -10,6 +10,21 @@ import DeleteSweepIcon from '@material-ui/icons/DeleteSweep';
 import {onFieldValidationChange, ValidationStatus} from '../helpers/validation';
 
 
+const validTemplates = ["%%CURRENCY_SYMBOL%%", "%%COUNTRY_NAME%%"];
+
+const getInvalidTemplateError = (text: string): string | null => {
```

Comment:
It's really time we had client-side testing in this project! This function could do with some tests.
I'll do this next week

Reply:
see latest commit!